src/components/roadmap/RoadmapKanban.tsx

```
@@ -0,0 +1,530 @@
+      return;
+    }
+    const sortedSuggestions = { ...localSuggestionsByStatus };
+    Object.keys(sortedSuggestions).forEach(statusId => {
+      sortedSuggestions[statusId] = [...sortedSuggestions[statusId]].sort((a, b) => {
+        switch (sortBy) {
+          case 'votes':
+            return b.votes_count - a.votes_count;
+          case 'date':
+            return new Date(b.submitted_at || 0).getTime() - new Date(a.submitted_at || 0).getTime();
+          case 'priority':
+            const priorityOrder = { critical: 0, high: 1, medium: 2, low: 3 };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)